resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders API
+  contact: {}
+  version: '1.0'
+  description: |
+    # Overview
+    Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS). 
+    Retrieve messages sent from customs offices of departure and destination.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+    variables: {}
+paths:
+  /customs/transits/movements/arrivals:
+    post:
+      tags:
+      - common-transit-convention-traders
+      summary: Send an arrival notification message
+      description: >
+        Notify the Customs Office of Destination that a transit movement has arrived at its final destination.
+
+        This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
+
+        A successful response will contain a URI and an arrival ID for the notification, which you can use to retrieve the message status.
+
+        It will also contain an embedded request ID, box ID and box name, which you can use to match this movement to any messages received from the Push Pull Notifications API.
+
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: SendAnArrivalNotificationMessage
+      parameters:
+          - $ref: '#/components/parameters/contentTypeHeader'
+      responses:
+        '202':
+          description: 'Accepted'
+          content:
+            application/json:
+              schema:
+                allOf:
+                - $ref: '#/components/schemas/CustomsTransitsMovementsArrivalsResponse'
+                - description: JSON payload
+                  example:
+                    _links:
+                      self:
+                        href: /customs/transits/movements/arrivals/1
+                    arrivalId: '1'
+                    messageType: IE007
+                    body: <CC007A>...</CC007A>
+                    _embedded:
+                      notifications:
+                        boxId: 1234-5678-abcd-def
+                        requestId: /customs/transits/movements/arrivals/1
+                        boxName: customs/transits##1.0##notificationUrl
+              example:
+                _links:
+                  self:
+                    href: /customs/transits/movements/arrivals/1
+                arrivalId: '1'
+                messageType: IE007
+                body: <CC007A>...</CC007A>
+                _embedded:
+                  notifications:
+                    boxId: 1234-5678-abcd-def
+                    requestId: /customs/transits/movements/arrivals/1
+                    boxName: customs/transits##1.0##notificationUrl

```

Comment:
I ran the Redocly openapi linter on the YAML file.

It flagged a warning that this API operation must have at least one `4XX` response.

Shouldn't this POST operation have a 400 response and probably a 406 response?